pkg/sql/catalog/internal/catkv/stored_catalog.go

```diff
@@ -83,7 +86,11 @@ type DescriptorValidationModeProvider interface {
 func MakeStoredCatalog(
 	cr CatalogReader, dvmp DescriptorValidationModeProvider, monitor *mon.BytesMonitor,
 ) StoredCatalog {
-	sc := StoredCatalog{CatalogReader: cr, DescriptorValidationModeProvider: dvmp}
+	sc := StoredCatalog{
+		CatalogReader:                    cr,
+		DescriptorValidationModeProvider: dvmp,
+		comments:                         NewCommentCache(),
```

Comment:
Nothing should be cached in `catkv`.

Reply:
Yeah...I'll change this name a bit. It's actually not a cache, it only mirrors what's in storage. I'll need to rename it a little bit. Maybe something like just 'Comments'.... Sorry about the bad and confusing naming.